sig-scheduling/README.md

```diff
@@ -59,11 +59,9 @@ The following [subprojects][subproject-definition] are owned by sig-scheduling:
 ### poseidon
 - **Owners:**
   - https://raw.githubusercontent.com/kubernetes-sigs/poseidon/master/OWNERS
-### scheduler
+### scheduler-plugins
```

Comment:
@Huang-Wei should this subproject be kept as `scheduler` - with https://raw.githubusercontent.com/kubernetes/kubernetes/master/cmd/kube-scheduler/OWNERS and https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/scheduler/OWNERS - and a new subproject be created for `scheduler-plugins` with https://raw.githubusercontent.com/kubernetes-sigs/scheduler-plugins/master/OWNERS?

I might be missing something, but with this change, don't we loose tracking of ownership of the k/k parts? :thinking:

Reply:
Ah! I realized that when `scheduler-plugins` was "added", it's just its OWNERs files was added under `scheduler`, not as a standalone section.

All right, I just updated this part to make `scheduler-plugins` a separate section. PTAL.

BTW: it's interesting we always list "scheduler" as a "subproject". I suppose this should be a "core" component/project :)

Reply:
> BTW: it's interesting we always list "scheduler" as a "subproject". I suppose this should be a "core" component/project :)

Yeah :upside_down_face: